guild/bsc-reth-snapshot.md

```diff
@@ -46,3 +48,13 @@ bsc-reth node \
     --http.api="eth, net, txpool, web3, rpc" \
     --full
 ```
+
+### ArchiveNode
+
```

Comment:
why fullnode and archive node share same start command? They should be different

Reply:
There's no `--full` for archive node